src/web/app/components/question-types/question-edit-details-form/num-scale-question-edit-details-form.component.html

```diff
@@ -10,7 +11,8 @@
   </div>
```

Comment:
Currently, the increment step input has no maximum cap. Similar to the other inputs, a big number may also cause problems in the backend. I understand that this input has 1 difference with the rest - which is it allows decimal input. However, I think it's a bit odd for this input to allow user inputs with more than 9 digits.

What we can do is:
- Add a `max="999999999"` to increment step input
- Create a new method called `restrictFloatInputLength` to restrict **decimal** inputs to 9 digits (inc. the decimal point). The current `restrictPointsLength` method does not support decimal inputs, since it performs `parseInt`. See below comment on my suggestions for implementing this new method
- Add a `(input)="restrictFloatInputLength($event, 'step')"` to increment step input